deployment-jvm/local-jvm/example-runner/execAndRun.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2018-2021 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+NAMESPACE=$1
```

Comment:
This script would have been refactored if this had been a successful exercise.
I've updated it so that the script names are not passed a arguments as they will not change.